src/cmd/manage_nsfs.js

```diff
@@ -410,7 +411,7 @@ async function fetch_account_data(argv, from_file) {
         data = JSON.parse(raw_data.toString());
     }
     if (action !== ACTIONS.LIST && action !== ACTIONS.STATUS) _validate_access_keys(argv);
-    if (action === ACTIONS.ADD || action === ACTIONS.STATUS) {
+    if (action === ACTIONS.ADD) {
```

Comment:
removing `ACTIONS.STATUS` will lead to an error when trying to do status based on the access key.
We should keep this